source/Nevermore/Querying/AST/ISelectColumns.cs

```diff
@@ -110,4 +111,53 @@ public SelectRowNumber(Over over, string alias)
         public string GenerateSql() => $"ROW_NUMBER() {over.GenerateSql()} AS {alias}";
         public override string ToString() => GenerateSql();
     }
+
+    public class SelectColumn : ISelectColumns
+    {
+        readonly string columnName;
+
+        public SelectColumn(string columnName)
+        {
+            this.columnName = columnName;
+        }
+
+        public bool AggregatesRows => false;
+
+        public string GenerateSql() => $"[{columnName}]";
+    }
+
+    public class SelectJsonValue : ISelectColumns
+    {
+        readonly string jsonPath;
+        readonly Type elementType;
+
+        public SelectJsonValue(string jsonPath, Type elementType)
+        {
+            this.jsonPath = jsonPath;
+            this.elementType = elementType;
+        }
+
+        public bool AggregatesRows => false;
+
+        public string GenerateSql() => $"CONVERT({GetDbType()}, JSON_VALUE([JSON], '{jsonPath}'))";
+
+        string GetDbType()
```

Comment:
This function already exists in `Nevermore/Querying/AST/Where.cs` (I'm assuming that's where you got it from).

It seems significant enough to extract into a static helper that we can re-use?

Reply:
I had actually already done this, but just hadn't pushed it 😁